src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,107 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int GROW_CONSTANT = 2;
+    private T[] arrayList;
```

Comment:
arraylist field of array list - can be a bit confusing, change to smth like values, elements etc